modules/ossm-release-2-6-4.adoc

```diff
@@ -0,0 +1,32 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-4_{context}"]
+= {SMProductName} version 2.6.4
+
+This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.4, and includes the following `ServiceMeshControlPlane` resource version updates: 2.6.4, 2.5.7, and 2.4.13.
+
+This release addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.14 and later.
+
+include::snippets/ossm-current-version-support-snippet.adoc[]
+
+The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource. The version of {SMProductShortName} automatically ensures a compatible version of Kiali.
```

Comment:
```suggestion
The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource. The version of {SMProductShortName} automatically ensures a compatible version of {KialiProduct}.
```
I'm not sure what the precedent is for referring to the Kiali Operator, but typically, the word "Operator" is included whenever an Operator is being referred to.

Reply:
@lahinson, I am ignoring this comment to maintain consistency with the previous releases. We use a similar template for the past z-streams.